src/main/java/com/fcgl/madrid/forum/model/InternalStatus.java

```diff
@@ -0,0 +1,33 @@
+package com.fcgl.madrid.forum.model;
+
+public class InternalStatus {
```

Comment:
Returning a POJO in your controller returns json to the frontend. This class has pretty much everything we need for error responses. But will probably want to modify or delete altogether depending on our use case